infra/feast-operator/internal/controller/services/services.go

```diff
@@ -94,6 +94,15 @@ func (feast *FeastServices) deployFeastServiceByType(feastType FeastServiceType)
 	if err := feast.createDeployment(feastType); err != nil {
 		return feast.setFeastServiceCondition(err, feastType)
 	}
+	if pvcCreate, shouldCreate := shouldCreatePvc(feast.FeatureStore, feastType); shouldCreate {
+		if err := feast.createPVC(pvcCreate, feastType); err != nil {
+			return feast.setFeastServiceCondition(err, feastType)
+		}
+	} else {
+		if err := feast.deleteOwnedFeastObj(feast.initPVC(feastType)); err != nil && !apierrors.IsNotFound(err) {
```

Comment:
no need for `IsNotFound` check here... we already `return nil` on this in the `deleteOwnedFeastObj` function
```suggestion
		if err := feast.deleteOwnedFeastObj(feast.initPVC(feastType)); err != nil {
```

Reply:
thinking about this more... do we even want to return an error here ? do we care if the delete doesn't succeed? is that worth halting the FeatureStore's reconciliation process or health of the stack? 
maybe we just `_ = feast.deleteOwnedFeastObj(feast.initPVC(feastType))` ??

thoughts?